tests/acceptance/bootstrap/bootstrap.php

```diff
@@ -0,0 +1,14 @@
+<?php
+$pathToCore = \getenv('PATH_TO_CORE');
```

Comment:
Codacy found an issue: [The use of function getenv() is discouraged](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5940619)